dipy/sims/voxel.py

```diff
@@ -298,16 +298,14 @@ def multi_tensor(gtab, mevals, S0=100, angles=[(0, 0), (90, 0)],
     sticks = _check_directions(angles)
 
     for i in range(len(fractions)):
-            S = S + fractions[i] * single_tensor(gtab, S0=S0, evals=mevals[i],
-                                                 evecs=all_tensor_evecs(
-                                                     sticks[i]), snr=None)
+        S = S + fractions[i] * single_tensor(gtab, S0=S0, evals=mevals[i],
+                                             evecs=all_tensor_evecs(sticks[i]), snr=None)
```

Comment:
Is this still <80 characters? 

If you need to, you can put a newline right before `snr`